src/lib/icon/icon.spec.ts

```diff
@@ -1,11 +1,11 @@
-import {inject, async, fakeAsync, tick, TestBed} from '@angular/core/testing';
-import {SafeResourceUrl, DomSanitizer} from '@angular/platform-browser';
-import {HttpClientTestingModule, HttpTestingController} from '@angular/common/http/testing';
-import {Component} from '@angular/core';
-import {MatIconModule} from './index';
-import {MatIconRegistry, getMatIconNoHttpProviderError} from './icon-registry';
-import {FAKE_SVGS} from './fake-svgs';
-import {wrappedErrorMessage} from '@angular/cdk/testing';
+import { inject, async, fakeAsync, tick, TestBed } from '@angular/core/testing';
+import { SafeResourceUrl, DomSanitizer } from '@angular/platform-browser';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { Component } from '@angular/core';
+import { MatIconModule } from './index';
+import { MatIconRegistry, getMatIconNoHttpProviderError } from './icon-registry';
+import { FAKE_SVGS } from './fake-svgs';
+import { wrappedErrorMessage } from '@angular/cdk/testing';
```

Comment:
Looks like your IDE added some spaces here and elsewhere. WebStorm lets you configure this, not sure about Code.